src/openloco/date.cpp

```diff
@@ -0,0 +1,469 @@
+#include "date.h"
+#include "interop/interop.hpp"
+#include <limits>
+#include <utility>
+
+using namespace openloco::interop;
+
+namespace openloco
+{
+    static loco_global<uint32_t, 0x00525E2C> _current_day;
+    static loco_global<uint16_t, 0x00525E30> _day_counter;
```

Comment:
`_day_counter` sounds like it counts days, whereas it seems to indicate the progress into a day instead.

Reply:
Yea I don't know what to really call it, any ideas?

Reply:
Maybe something like `_day_progression`?

Reply:
ok